Core/scripts/dirac-install.py

```diff
@@ -2319,8 +2319,9 @@ def createBashrcForDiracOS():
       lines = ['# DIRAC bashrc file, used by service and agent run scripts to set environment',
                'export PYTHONUNBUFFERED=yes',
                'export PYTHONOPTIMIZE=x',
-               '[ -z "$DIRACOS" ] && export DIRACOS=%s/diracos' % proPath,
-               '. %s/diracos/diracosrc' % proPath]
+               '[ -z "$DIRAC" ] && export DIRAC=%s' % proPath,
+               '[ -z "$DIRACOS" ] && export DIRACOS=$DIRAC/diracos',
+               '. $DIRAC/diracos/diracosrc']
```

Comment:
There was some discussions on slack that one might want to set DIRACOS to some other path before sourcing the bashrc (if I understand correctly?) In that case shouldn't this line be `'. $DIRACOS/diracosrc']` ?

Reply:
Yes it's better. Fixed.